### PR TITLE
finished database and server

### PR DESCRIPTION
**Pull request by ykent4**

### CODE REVIEW

**Comment by ykent4** [comment]
@AndrewPomo  Hi Andrew, please review my code. The finished parts are the following:
- setups for mongodb, webpack, server, jest, react
- database
- server

File: database/Image.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    living_room: String,
+    bedroom: String, 
+    dining_room: String,
+    den: String
+  }
+});
+
+const Image = mongoose.model('Image', imageSchema);
+
+module.exports.Image = Image;
```
Comment (AndrewPomo) [inline database/Image.js:27]: @ykent4 AirBnB styleguide would have you put a new line at the end of this file.

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+
```
Comment (AndrewPomo) [inline database/seed.js:1]: @ykent4 Delete leading line

File: database/seed.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+//create document for each image
+var imageDataCreater = function () {
+
+  var resultArr = [];
+
+  for(var i = 1; i < 101; i++){
+    resultArr.push({
+        title: 'Relax in the comfort of '+ faker.company.bsBuzz,
```
Comment (AndrewPomo) [inline database/seed.js:16]: @ykent4 What is `bsBuzz`? I did a quick google through Faker docs and the only example I found was "maximize."

That would leave the title as "Relax in the comfort of maximize," which I'm not sure indicates that bsBuzz is a great fit. Maybe I just found a bad example, though. Lemme know!

File: database/seed.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        });
+
+    }
+
+  return resultArr;
+}; 
+
+
+var sampleImagesArr = imageDataCreater();
+
```
Comment (AndrewPomo) [inline database/seed.js:40]: @ykent4 I believe `insertSampleImages` is the way to go here, so go ahead and delete the commented out code.

**Review by AndrewPomo — COMMENTED** [review]
Mix of suggestions and questions here. Let me know if I can help with anything.

File: database/Image.js (hunk 1 above)
Comment (ykent4) [reply database/Image.js:27]: thanks! changed

File: database/seed.js (hunk 2 above)
Comment (ykent4) [reply database/seed.js:1]: thanks! changed

File: database/seed.js (hunk 3 above)
Comment (ykent4) [reply database/seed.js:16]: return faker.random.arrayElement(faker.definitions.company.bs_verb);

The above return statement is from bsBuzz in the faker library. It seems to return a verb. Thanks for pointing out how that would be a very awkward string. I'm changing this to bsNoun.

File: database/seed.js (hunk 4 above)
Comment (ykent4) [reply database/seed.js:40]: thanks! changed

**Pull request merged**